src/bacnet/basic/object/bv.c

```diff
@@ -1367,6 +1367,90 @@ static struct object_data *Binary_Value_Object_Index(int index)
 {
     return Keylist_Data_Index(Object_List, index);
 }
+
+/**
+ * For a given object instance-number, returns the event_enable property value
+ *
+ * @param  object_instance - object-instance number of the object
+ *
+ * @return  event_enable property value
+ */
+uint32_t Binary_Value_Event_Enable(uint32_t object_instance)
+{
+    uint32_t event_enable = 0;
+    struct object_data *pObject = Binary_Value_Object(object_instance);
+
+    if (pObject) {
+        event_enable = pObject->Event_Enable;
+    }
+
+    return event_enable;
+}
+
+/**
+ * For a given object instance-number, sets the event_enable property value
+ *
+ * @param object_instance - object-instance number of the object
+ * @param event_enable - event_enable property value
+ *
+ * @return true if the event_enable property value was set
+ */
+bool Binary_Value_Event_Enable_Set(uint32_t object_instance, uint32_t event_enable)
+{
+    bool status = false;
+    struct object_data *pObject = Binary_Value_Object(object_instance);
+
+    if (pObject) {
+      if(!(event_enable & ~(EVENT_ENABLE_TO_OFFNORMAL | EVENT_ENABLE_TO_FAULT | EVENT_ENABLE_TO_NORMAL))) {
+        pObject->Event_Enable = event_enable;
+        status = true;
+      }
+    }
+
+    return status;
+}
+
+/**
+ * For a given object instance-number, returns the notify_type property value
+ *
+ * @param  object_instance - object-instance number of the object
+ *
+ * @return  notify_type property value
+ */
+BACNET_NOTIFY_TYPE Binary_Value_Notify_Type(uint32_t object_instance)
+{
+    BACNET_NOTIFY_TYPE notify_type = NOTIFY_EVENT;
+    struct object_data *pObject = Binary_Value_Object(object_instance);
+
+    if (pObject) {
+        notify_type = pObject->Notify_Type;
+    }
+
+    return notify_type;
+}
+
+/**
+ * For a given object instance-number, sets the notify_type property value
+ *
+ * @param object_instance - object-instance number of the object
+ * @param notify_type - notify_type property value
+ *
+ * @return true if the notify_type property value was set
+ */
+bool Binary_Value_Notify_Type_Set(uint32_t object_instance, BACNET_NOTIFY_TYPE notify_type)
+{
+    bool status = false;
+    struct object_data *pObject = Binary_Value_Object(object_instance);
+
+    if (pObject) {
+      if((notify_type == NOTIFY_EVENT) || (notify_type == NOTIFY_ALARM)) {
```

Comment:
Add a @note in header as to the setter only works for these 2 guys and not for whatever other guy